scripts/zones/Arrapago_Reef/MobIDs.lua

```diff
@@ -0,0 +1,6 @@
+-- Medusa
+Medusa    = 16998862;
+ExonOne   = 16998863;
```

Comment:
@nesstea No need for this. Can use relative IDs since they are all next to each other.
Instead of:
```lua
function onMobEngaged(mob, target)
    target:showText(mob, MEDUSA_ENGAGE);
    SpawnMob(ExonOne,   180):updateEnmity(target);
    SpawnMob(ExonTwo,   180):updateEnmity(target);
    SpawnMob(ExonThree, 180):updateEnmity(target);
    SpawnMob(ExonFour,  180):updateEnmity(target);
end;
```

you'd use:
```lua
function onMobEngaged(mob, target)
    target:showText(mob, MEDUSA_ENGAGE);
    SpawnMob(mob:getID()+1, 180):updateEnmity(target);
    SpawnMob(mob:getID()+2, 180):updateEnmity(target);
    SpawnMob(mob:getID()+3, 180):updateEnmity(target);
    SpawnMob(mob:getID()+4, 180):updateEnmity(target);
end;
```

Groupings like this are stable through client ID shifts: if medusa gets moved by one her helpers get moved by the same amount and stay the same relative to each other, every single time.